src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -64,7 +62,7 @@ public SonarWebService(IDownloader downloader, string server, ILogger logger)
             var qualityProfileKey = await DoLogExceptions(async () =>
             {
                 var result = await downloader.TryDownloadIfExists(uri);
-                string contents = result.Item2;
+                var contents = result.Item2;
```

Comment:
Let's deconstruct it instead
```
var (success, result) = await downloader.TryDownloadIfExists(uri);
```

Reply:
[CS1061] 'Tuple<bool, string>' does not contain a definition for 'Deconstruct' and no accessible extension method 'Deconstruct' accepting a first argument of type 'Tuple<bool, string>' could be found (are you missing a using directive or an assembly reference?)

Reply:
😿

Reply:
😒